decidim-conferences/app/views/decidim/conferences/admin/conference_speakers/index.html.erb

```diff
@@ -25,48 +25,77 @@
-              <% end %>
+            <td class="table-list__actions" data-label="<%= t("models.conference_speaker.fields.actions", scope: "decidim.admin") %>">
+              <button type="button" data-component="dropdown" data-target="actions-conference-speaker-<%= speaker.id %>" aria-label="<%= t("decidim.admin.actions.actions_label", resource: speaker_presenter.name) %>">
+                <%= icon "more-fill" %>
+              </button>
+
+              <div class="inline-block relative">
+                <ul id="actions-conference-speaker-<%= speaker.id %>" class="dropdown dropdown__action" aria-hidden="true">
+                  <% if allowed_to? :update, :conference_speaker, speaker: speaker %>
+                    <li class="dropdown__item">
+                      <%= link_to edit_conference_speaker_path(current_conference, speaker), class: "dropdown__button" do %>
+                        <%= icon "pencil-line" %>
+                        <%= t("actions.edit", scope: "decidim.admin") %>
+                      <% end %>
+                    </li>
+                  <% end %>
+
+                  <hr>
+
+                  <% if allowed_to? :update, :conference_speaker, speaker: speaker %>
+                    <% if speaker.published? %>
+                      <li class="dropdown__item">
+                        <%= link_to unpublish_conference_speaker_path(current_conference, speaker.id), method: :put, class: "dropdown__button" do %>
+                          <%= icon "close-circle-line" %>
+                          <%= t("actions.unpublish", scope: "decidim.admin") %>
+                        <% end %>
+                      </li>
+                    <% else %>
+                      <li class="dropdown__item">
+                        <%= link_to publish_conference_speaker_path(current_conference, speaker.id), method: :put, class: "dropdown__button" do %>
+                          <%= icon "check-line" %>
+                          <%= t("actions.publish", scope: "decidim.admin") %>
+                        <% end %>
+                      </li>
+                    <% end %>
+                  <% end %>
 
-              <% if allowed_to? :update, :conference_speaker, speaker: speaker %>
-                <% if speaker.published? %>
-                  <%= icon_link_to "close-circle-line", unpublish_conference_speaker_path(current_conference, speaker.id), t("actions.unpublish", scope: "decidim.admin"), method: :put, class: "action-icon--unpublish" %>
-                <% else %>
-                  <%= icon_link_to "check-line", publish_conference_speaker_path(current_conference, speaker.id), t("actions.publish", scope: "decidim.admin"), method: :put, class: "action-icon--publish" %>
-                <% end %>
-              <% else %>
-                <span class="action-space icon"></span>
-              <% end %>
+                  <hr>
```

Comment:
Let's move the HR inside the above permission check

Reply:
Fixed on 2edcdcb5ea1d34de68fb35c5d3038169988dc759